### DISCUSSION TITLE
MCBBS寄了，哪时候能找到新的国内下载源

### DISCUSSION THREAD

**Discussion by LxyFreely** [Meloong-Git/PCL#3323] [讨论]
直连微软是真的慢啊

**Comment by Light-Beacon** [comment]
下载源这种东西完全就是在做慈善，巨能烧钱还不是很能捞好处。感觉不太可能有新的下载源

**Reply by Deep-Dark-Forest** [reply]
能捞一点点好处，至少有很少的人给他们赞助（

**Reply by Light-Beacon** [reply]
> 能捞一点点好处，至少有很少的人给他们赞助（

赞助也抵不上开销

**Comment by Ad-closeNN** [comment]
挂`7根木棍做的东西`是不是能缓解一些

**Reply by Deep-Dark-Forest** [reply]
是

**Comment by LxyFreely** [comment]
能用MCLAPI下载源不

**Reply by wuliaodexiaoluo** [reply]
BMCLI API 没说不能用但是不稳定

**Comment by Pigeon0v0** [comment]
现在 BMCLAPI 很能扛了

**Reply by wuliaodexiaoluo** [reply]
也不算特别能抗，MCBBS离线马上超载871.85%

**Reply by zkitefly** [reply]
> 也不算特别能抗，MCBBS离线马上超载871.85%

那是以前，现在都没过 100%

**Reply by wuliaodexiaoluo** [reply]
等哪天Mojang API 维护。。。流量全部路由到BMCLAPI，直接爆炸

**Reply by allMagicNB** [reply]
> 等哪天Mojang API 维护。。。流量全部路由到BMCLAPI，直接爆炸

哪有那么容易炸啊，你不去看看有几个节点

**Reply by wuliaodexiaoluo** [reply]
但并非不可能，CDN会炸吗，当然会，是服务器就有宕机的风险，只不过热备冷备CDN分布式节点和CLB可以减少风险出现的可能性

**Reply by wuliaodexiaoluo** [reply]
~概率永远不会按3050这样计，发生了就是100%，没发生就是0%~

**Reply by Pigeon0v0** [reply]
> 等哪天Mojang API 维护。。。流量全部路由到BMCLAPI，直接爆炸

国内靠 Mojang 下的本来就没多少...

而且你不想想人家这个可用性
